OSSM-5917: Add text indicating that imported service requires alias (#71852)
openshift/openshift-docs: modules/ossm-federation-config-import.adoc

  importRules:
  - type: NameSelector
    importAsLocal:

> |Set to `true` to aggregate remote endpoint with local services. When `true` services are imported as `<name>.<namespace>.svc.cluster.local`. When `true` an alias is required. When `false` no alias is required. 

Author: Done

> |Set to `true` to aggregate remote endpoint with local services. When `true` services will be imported as `<name>.<namespace>.svc.cluster.local`. When 'true' an alias is required. When `false` no alias is required. 

Reviewer: ```
'true'
```
@rh-tokeefe  probably typo. Apostrophes ('') are used instead of Grave accents (``) so that `true` is not highlighted as the others in that section ( https://71852--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-federation#ossm-federation-config-import_federation )

Author: Great catch @mkralik3 ! PR has been updated. I will squash the PR after you review and give it LGTM.

> |Set to `true` to aggregate remote endpoint with local services. When `true` services will be imported as `<name>.<namespace>.svc.cluster.local`. When `true` an alias is required. When `false` no alias is required. 

Reviewer 2: I'd just suggest trying to write the second sentence in present tense per the ISG's [guidance on verb tense](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense):
```suggestion
|Set to `true` to aggregate remote endpoint with local services. When `true` services are imported as `<name>.<namespace>.svc.cluster.local`. When `true` an alias is required. When `false` no alias is required. 
```

Author: Made this edit.
